server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala

```diff
@@ -486,6 +488,51 @@ trait WebSocketContract {
       .contains(stateChange)
   }
 
+  @Test
+  @Timeout(180)
+  def shouldPushChangesToDelegatedUser(server: GuiceJamesServer): Unit = {
```

Comment:
Please add a test ensuring that if alice delegates her mailbox to bob then alice can still receive her notifications.